transforms/acceptance/tracking_page_event_migration.js

```diff
@@ -0,0 +1,54 @@
+const {
+  addImport
+} = require('../../utils/imports');
+
+function substituteEvent(j, code, eventName) {
+  let needsImport = false;
+  code = j(code).find(j.CallExpression, {
+    callee: {
+      type: 'MemberExpression',
+      object: { type: 'ThisExpression' },
+      property: { type: 'Identifier', name: eventName }
+    }
+  })
+  .forEach(path => {
+    needsImport = true;
+    j(path).replaceWith(
+        j.callExpression(
+          j.identifier(path.node.callee.property.name),
+          path.node.arguments
+        )
+    );
+  }).closest(j.CallExpression, {
+    callee: { name: 'module' }
+  }).forEach(path => {
+    let setupExists = j(path).find(j.Identifier, { name: 'setupIntercomEventService' }).length !== 0;
+    if (!setupExists) {
+      path.value.arguments[1].body.body.unshift("setupIntercomEventService(hooks);");
+    }
+  }).toSource();
+
+  if (needsImport) {
+    code = addImport(j, code, eventName, 'embercom/tests/helpers/services/intercom-event');
```

Comment:
We could call these where we do `needsImport = true;` instead as the function on't doubly import anything

Reply:
Yeah but imagine a file having 10 test cases where `needsImport` is set to true, we don't want to modify the file 10 times instead of just once at the end. Just a performance thing.

Reply:
True, happy to leave it as is then 👍